examples/opf/experiments/anomaly/temporal/hotgym/description.py

```diff
@@ -20,13 +20,11 @@
 # ----------------------------------------------------------------------
 
 """
-Template file used by the OPF Experiment Generator to generate the actual
-description.py file by replacing $XXXXXXXX tokens with desired values.
-
-This description.py file was generated by:
-'~/nupic/eng/lib/python2.6/site-packages/nupic/frameworks/opf/expGenerator/ExpGenerator.py'
+Anomaly hotgym example
 """
 
+from nupic.algorithms.anomaly import Anomaly
```

Comment:
`Anomaly` imported here, but not used. Please run the changed files through pylint with nupic pylint config and fix pylint findings related to your changes.